setup.py

```diff
@@ -25,6 +25,7 @@
     'mssql': ['pyodbc'],
     'mysql': ['PyMySQL>=0.8.0'],
     'odata': auth_deps + ['tctc_odata'],
+    'odbc': ['pyodbc', 'psycopg2'],
```

Comment:
Is `psycopg2` the postgres adaptor for odbc?
As it seems only used by tests, maybe it's a better idea to put it in `requirements-testing.txt`, so it does not get installed in production